src/library/vtkF3DAlembicImporter.h

```diff
@@ -113,7 +111,8 @@ class vtkF3DAlembicImporter : public vtkImporter
   vtkF3DAlembicImporter(const vtkF3DAlembicImporter&) = delete;
   void operator=(const vtkF3DAlembicImporter&) = delete;
 
-  vtkNew<vtkF3DAlembicImporterInternal> Internals;
+  class vtkInternals;
+  std::unique_ptr<vtkInternals> Internals;
```

Comment:
FYI @nyue : sorry about my commnents on your PR. Here is the right way to do it. More info here: https://discourse.vtk.org/t/pimpl-management-good-practice/7582/16